src/Backend/Modules/Faq/Domain/Question/QuestionRepository.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Backend\Modules\Faq\Domain\Question;
+
+use Backend\Modules\Faq\Domain\Category\Category;
+use Common\Core\Model;
+use Doctrine\ORM\EntityRepository;
+use Common\Locale;
+use Doctrine\ORM\NoResultException;
+
+final class QuestionRepository extends EntityRepository
+{
+    /**
+     * @param string $url
+     * @param Locale $locale
+     *
+     * @return Question
+     *
+     * @throws NoResultException When no result is found
+     */
+    public function findOneByUrl(string $url, Locale $locale): Question
+    {
+        return $this
+            ->createQueryBuilder('q')
+            ->innerJoin('q.meta', 'm')
+            ->andWhere('m.url = :url')
+            ->andWhere('q.locale = :locale')
+            ->setParameter(':url', $url)
+            ->setParameter(':locale', $locale)
+            ->getQuery()
+            ->getSingleResult()
+        ;
+    }
+
+    public function findByCategory(
+        Category $category,
+        Locale $locale,
+        ?int $limit = null,
```

Comment:
why this `?` again, it is completely unnecessary?

Reply:
Fixed e9534a7d6